assets/javascripts/text_blocks.js

```diff
@@ -35,9 +35,16 @@ var TextBlocks = {
   },
 
   reload: function(e){
+    debugger
```

Comment:
@dkastl @mopinfish  
Sorry, I forgot to delete this line...
I will delete this line on `main` branch directly.

Reply:
Well, it seems to impossible to push `main` branch directly, so I will make PR to delete this line...